db/upgrade.php

```diff
@@ -189,5 +189,25 @@ function xmldb_local_augmented_teacher_upgrade($oldversion) {
         // Augmented_teacher savepoint reached.
         upgrade_plugin_savepoint(true, 2019010800, 'local', 'augmented_teacher');
     }
+
+    if ($oldversion < 2022073000) {
+        $table = new xmldb_table('local_augmented_teacher_sent');
+
+        $table->add_field('id', XMLDB_TYPE_INTEGER, '18', null, XMLDB_NOTNULL, XMLDB_SEQUENCE, null);
+        $table->add_field('userid', XMLDB_TYPE_INTEGER, '18', null, XMLDB_NOTNULL, null, null);
+        $table->add_field('reminderid', XMLDB_TYPE_INTEGER, '18', null, XMLDB_NOTNULL, null, null);
+        $table->add_field('timesent', XMLDB_TYPE_INTEGER, '18', null, XMLDB_NOTNULL, null, '0');
+
+        $table->add_key('id', XMLDB_KEY_PRIMARY, ['id']);
+        $table->add_index('userid_reminderid_ix', XMLDB_INDEX_NOTUNIQUE, ['userid', 'reminderid']);
```

Comment:
Same as previous comment.

Reply:
A user can receive the same reminder multiple times so it should not be unique